pkg/antctl/raw/traceflow/command.go

```diff
@@ -36,22 +37,16 @@ import (
 
 	"antrea.io/antrea/pkg/antctl/raw"
 	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	antrea "antrea.io/antrea/pkg/client/clientset/versioned"
 )
 
 const defaultTimeout time.Duration = time.Second * 10
 
 var (
-	Command *cobra.Command
-	option  = &struct {
-		source      string
-		destination string
-		outputType  string
-		flow        string
-		liveTraffic bool
-		droppedOnly bool
-		timeout     time.Duration
-		nowait      bool
-	}{}
+	Command      *cobra.Command
+	getClients   = getk8sClient
+	newTraceflow = getNewTraceflow
+	getTFName    = getTfName
```

Comment:
Why do you need to fake this method? it just return a traceflow Name.

Reply:
I need to fake it because it returns traceflow name by appending 8 random characters at the end of prefix(<srcName-to-dstName>). So, i cannot predict the actual traceflow name beforehand in my expected traceflow.

Reply:
You can ignore comparing the traceflow name, for example, check expected spec only.